### ISSUE TITLE
Back link changed from blue to grey

### ISSUE THREAD

**Issue by JuliannaR** [cds-snc/covid-shield-mobile#478]
The ‘Back’ link in onboarding and in ‘How it works’  are all faded grey rather than blue like other hyperlinks. 
This should be changed to the blue found on the cancel links.

![Image from iOS (12)](https://user-images.githubusercontent.com/35537308/86786481-0f8cb980-c032-11ea-8212-4b233c86bd48.png)
